lib/exchange/jwt.js

```diff
@@ -115,7 +116,7 @@ module.exports = function jwtBearer(options, issue) {
       , separator = '.';
 
     if (!jwtBearer) {
-      return next(new AuthorizationError('missing assertion parameter', 'invalid_request'));
+      return next(new AuthorizationError(g.f('missing assertion parameter', 'invalid_request')));
```

Comment:
Bracket in the wrong place.
g.f('missing assertion parameter'), 'invalid_request'`